verify/verifier.go

```diff
@@ -43,13 +54,29 @@ func (c Config) MarshalJSON() ([]byte, error) {
 	return json.Marshal(aux)
 }
 
+// Verifier verifies the integrity of a blob and its certificate. There are 3 main categories of verification:
+//  1. Blob: properties localized to a single blob, most specifically its commitment
+//  2. Batch: properties wrt the EigenDA network: signatures, quorum thresholds, onchain inclusion, etc.
+//  3. Rollup: properties wrt the rollup chain: block at which the cert was included in the batcher's inbox
+//     (make sure its not too stale trying to game fraud proof windows)
+//
 // TODO: right now verification and confirmation depth are tightly coupled. we should decouple them
 type Verifier struct {
```

Comment:
I'm curious why you are adding the recency check to this V1 logic.

Wouldn't it be ok to leave V1 entirely as-is, and *only* worry about the recency check for V2?

Reply:
yes I actually totally agree... I started this PR loong ago before we had v2. So just brought it back to life with me. I think it might actually make sense to just remove the V1 logic entirely to keep proxy simple...

Reply:
Ok pulled the trigger. Completely removed v1 verification, so moved this config to v2ClientCfg. Can't link the actual commit because I chnaged/cleanup a lot of stuff..... think you'll need to re-review the whole thing (don't think it'll be that bad).